core/src/main/java/org/mule/runtime/core/internal/config/FeatureFlaggingServiceBuilder.java

```diff
@@ -31,6 +33,7 @@
  *
  * @see FeatureFlaggingService
  * @see org.mule.runtime.core.api.config.FeatureFlaggingRegistry
+ *
  * @since 4.4.0
  */
 public final class FeatureFlaggingServiceBuilder {
```

Comment:
can this class be moved to the new feature-management module?

Reply:
No, it can. It has dependencies on classes from core. MuleContext for example. I would have a cyclic dependency.

Reply:
create a jira ticket to track this

Reply:
Created ticket MULE-19954.